src/components/Header/MenuLink/MenuLink.module.css

```
@@ -16,3 +21,38 @@
 .menu__link--selected {
   border-bottom: 3px solid #f3e049;
 }
+
+.connexion{
+  padding-top: 20px;
+  padding-left: 20px;
+}
+
+.connexion>a>a{
+  display: flex;
+}
+
+.powerSetting>a, .profilSetting>a{
+  display: flex;
+}
+
+.powerSetting img, .connexion img{
+  margin-right: 5px;
+  width: 20px;
+}
+
+
+@media screen and (min-width: 991px){
+  .connexion{
+    display : none;
```

Bon, là, par contre, y'en a un de trop 😄 

```suggestion
    display: none;
```